docs/howto/add_activable_element_from_selection.md

```diff
@@ -0,0 +1,69 @@
+# How-to: Add activable element in a selection form
+
+The Hybrid Platform UI application provides a custom element called `ez-selection-activate-element`. This component gives the possibility to easily implement activable
+elements (like buttons) that will be enabled and can triggers actions only if some elements in a
+list are selected.
+The selection is done with the help of checkboxes or radio buttons. Some activable elements
+need at least one or more element checked to be enabled, but others will need one and only one
```

Comment:
can only be enabled in at least one or more elements are checked. Others will need one and only one